_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "WebOké",
+    "link": "https://weboke.nl/",
+    "category": "full",
+    "tutorial": "https://www.weboke.nl/knowledgebase/80/Gratis-Lets-encrypt-certificaat-instellen.html",
+    "announcement": "https://weboke.nl/webhosting",
+    "plan": "",
+    "reviewed": "",
+    "note": "All webhosting plans that WebOké offers inculde free Let's Encrypt certificates which can be installed within 1 minute."
```

```suggestion
    "note": ""
```
No need to add this as it's the standard definition of "partial" support; you may instead want to say "Website is in Dutch", though.

---

Thanks a lot for the feedback! I have changed this now, i think..